cumulusci/cumulusci.yml

```diff
@@ -233,6 +233,13 @@ tasks:
         options:
             suites: tests
         group: Robot Framework
+    robot_documentation:
+        description: Generates documentation for project keyword files
+        class_path: cumulusci.tasks.robotframework.RobotDoc
+        options:
+            outputdir: docs
+            title: $project_config.project__package__name
+        group: Robot Framework
```

Comment:
@boakley this task definition needs to be updated

Reply:
Done